src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -133,6 +135,28 @@
  *        .listIterator()
  *        .forEachRemaining(System.out::print); // OK
  * </pre>
+ * <p>
+ *     To Configure the check to restrict the use of whitespace before
+ *     LABELED_STAT tokens:
```

Comment:
I see no reason for separate example. This token is enabled by default, please add it to default example

Reply:
Done.